pkg/cvo/internal/operatorstatus.go

```diff
@@ -220,21 +226,27 @@ func waitForOperatorStatusToBeDone(ctx context.Context, interval time.Duration,
 				message = fmt.Sprintf("Cluster operator %s is reporting a failure: %s", actual.Name, condition.Message)
 			}
 			lastErr = &payload.UpdateError{
-				Nested:  errors.New(lowerFirst(message)),
-				Reason:  "ClusterOperatorDegraded",
-				Message: message,
-				Name:    actual.Name,
+				Nested:       errors.New(lowerFirst(message)),
+				UpdateEffect: payload.UpdateEffectFail,
+				Reason:       "ClusterOperatorDegraded",
```

Comment:
I still think we should ignore ClusterOperator Degraded entirely (#482), but as long as we keep it, can it at least be `UpdateEffectFailAfterInterval` or `UpdateEffectNone` (I prefer the latter, but am ok compromising on the former ;)?

Reply:
I can soften this as you're suggesting but shouldn't this also only apply if `Available=True`. So overall it's really only `ClusterOperatorDegraded` if `Available=True` && `Degraded=True`? Otherwise I'll drop through and handle it below resulting in 'ClusterOperatorNotAvailable` ...

Reply:
Changed per above comment

Reply:
Yeah, `Available=False` should be `UpdateEffectFail` regardless of `Degraded`.  dee13735 looks good to me.